### PR TITLE
HTTPizza/2.0

### PR DESCRIPTION
**Pull request by rien**
Event post of HTTPizza/2.0

CC: @rubenverborgh

### CODE REVIEW

**Comment by rien** [comment]
This is the poster:
HTTPizza
The link https://pizza.zeus.gent and https://:pizza:.zeus.gent will work once this PR is accepted.

**Comment by werthen** [comment]
Check out the preview on https://316.pr.zeus.gent/

File: content/events/18-19/httpizza2.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+title: HTTPizza/2.0
+image: 'https://zeus.ugent.be/zeuswpi/O9PhQ5qJ.png'
+description: Hands-on HTTP ervaring tussen de slices pizza door.
```
Comment (niknetniko) [inline content/events/18-19/httpizza2.md]: HTTP ervaring → HTTP-ervaring of ervaring met HTTP

File: content/events/18-19/httpizza2.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+location: Auditorium A3, Gebouw S9, Campus Sterre
+locationlink: S9 Gent
+facebook: 'https://www.facebook.com/events/326902994607510/'
+end: 21-02-2019 22:00
+---
+
+Na het succes van HTTPizza in 2017 komt nu HTTPizza/2.0. [Zeus WPI](https://zeus.gent/) en [IDLab](http://idlab.technology/)
+organiseren samen deze ludieke netwerking-hackathon: Je krijgt een aantal
+uitdagingen die je moet oplossen door het schrijven van een client voor ons
+zelfgemaakte 'HTTPizza' protocol. Voor iedere opgeloste challenge krijg je een
```
Comment (niknetniko) [inline content/events/18-19/httpizza2.md]: 'HTTPizza' protocol → 'HTTPizza'-protocol

**Comment by RubenVerborgh** [comment]
> https://🍕.zeus.gent

Sweet! Was traveling, but good work!

**Pull request merged**